WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -220,10 +221,29 @@ private extension SupportTableViewController {
                 // if the value changed.
                 WPAnalytics.track(.supportIdentitySet)
                 self.reloadViewModel()
+                self.checkForAutomatticEmail()
             }
         }
     }
 
+    /// Zendesk does not allow agents to submit tickets, and displays a 'Message failed to send' error upon attempt.
+    /// If the user email address is a8c, display a warning.
+    ///
+    func checkForAutomatticEmail() {
+        guard let email = ZendeskUtils.userSupportEmail(),
+            (Constants.automatticEmails.first { email.contains($0) == true }) != nil else {
```

Comment:
You could drop the `== true` and it will still work, fwiw.

Reply:
Well, you would be correct, and tis been removed. Thank you!